src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,7 +6,7 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable {
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
```

Comment:
I am still very new to java. I know the difference between public and private methods and variables. But could you enlighten me on the effect of making a class private/public? If it is private, does it mean other classes cannot create an instance of the private class?